framework/auth/cas.py

```diff
@@ -149,6 +149,20 @@ def _parse_profile(self, raw):
             resp.attributes[attribute] = data['attributes'][attribute]
         return resp
 
+    def revoke_application_tokens(self, client_id, client_secret):
+        """Revoke all tokens associated with a given CAS client"""
+        url = furl.furl(self.BASE_URL)
+        url.path.segments.extend(('oauth2', 'revoke'))
+        url.args['client_id'] = client_id
+        url.args['client_secret'] = client_secret
```

Comment:
Ping @icereval for review. Is putting the client secret in the query params the preferred approach?